tests/test_collection.py

```diff
@@ -15,14 +15,16 @@
 class CollectionTest(TfsCollection):
     file = Tfs("file_{}.tfs")
     nofile = Tfs("nofile_{}.tfs")
-    filex = Tfs("file_x.tfs", two_planes=False)
+    filex = Tfs("file_X.tfs", two_planes=False)
```

Comment:
https://github.com/pylhc/tfs/issues/47#issuecomment-696057755

Will this only work now on files with _X, _Y?

Reply:
After our discussion with @lmalina, I have done the proper implementation in 44dff0d and will be writing a short comment on issue #47 to clarify exactly what we meant and what we agreed on implementing.